### ISSUE TITLE
Can this library allow for displaying dash '-'

### ISSUE THREAD

**Issue by arfan** [JackHurst0/seven-segment-display#3]
Can this library allow for displaying dash '-' ? as it is also a possible to display in a 7 segment, and also maybe we can turn on and off any segment in the seven-segment so that we can further customize what we can display in a 7 segment.

regards.

**Comment by JackHurst0** [comment]
Yes this is a good idea, I will look into adding this feature for you.
